modules/ixBidAdapter.js

```diff
@@ -948,58 +1011,43 @@ export const spec = {
    * @return {object}                  Info describing the request to the server.
    */
   buildRequests: function (validBidRequests, bidderRequest) {
-    let reqs = [];
-    let bannerImps = {};
-    let videoImps = {};
-    let validBidRequest = null;
-
-    // To capture the missing sizes i.e not configured for ix
-    let missingBannerSizes = {};
+    const reqs = []; // Stores banner + video requests
+    const bannerImps = {}; // Stores created banner impressions
+    const videoImps = {}; // Stores created video impressions
+    const multiFormatAdUnits = {}; // Stores references identified multi-format adUnits
+    const missingBannerSizes = {}; // To capture the missing sizes i.e not configured for ix
+
+    // Step 1: Create impresssions from IX params
+    validBidRequests.forEach((validBidRequest) => {
+      const adUnitMediaTypes = Object.keys(utils.deepAccess(validBidRequest, 'mediaTypes', {}))
+
+      switch (detectParamsType(validBidRequest)) {
+        case BANNER:
+          createBannerImps(validBidRequest, missingBannerSizes, bannerImps);
+          break;
+        case VIDEO:
+          createVideoImps(validBidRequest, videoImps)
+          break;
+      }
 
-    const DEFAULT_IX_CONFIG = {
-      detectMissingSizes: true,
-    };
+      if (adUnitMediaTypes.includes(BANNER) && adUnitMediaTypes.includes(VIDEO)) {
+        multiFormatAdUnits[validBidRequest.transactionId] = validBidRequest;
+      }
+    });
```

Comment:
Hey @lksharma . Since we still are using ie11 for testing and we support it we aren't able to use .includes right now (https://caniuse.com/array-includes). However, in the coming months with Prebid 6.0 release we are dropping ie support so should be able to use this in the near future